sdk/avs/Microsoft.Azure.Management.Avs/src/Microsoft.Azure.Management.Avs.csproj

```
@@ -0,0 +1,18 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <!-- Please do not move/edit code below this line -->
+  <Import Project="$(RepoEngPath)/mgmt/AzSdk.reference.props" />
+  <!-- Please do not move/edit code below this line -->
+
+  <PropertyGroup>
+    <PackageId>Microsoft.Azure.Management.Avs</PackageId>
+    <Description>Provides developers with libraries for Azure VMware Solution.</Description>
+    <Version>0.1.0</Version>
```

```suggestion
    <Version>1.0.0-preview.1</Version>
```
The first version of a new module is typically `1.0.0-preview.1` according to our versioning guide: https://github.com/Azure/azure-sdk-for-net/blob/master/doc/dev/Versioning.md
Then it goes to 1.0.0-preview.2 and finally 1.0.0